draft-retana-idr-bgp-quic-02.xml

```diff
@@ -364,7 +364,7 @@
             <tr>
               <td>68</td>
               <td>Multisession BGP Capability</td>
-              <td>draft-ietf-idr-bgp- multisession</td>
```

Comment:
These splits are, sadly, deliberate.  Without them, the ascii form of the table doesn't render in a pleasing way.

If you have thoughts about how to influence column rendering for the generated table to split on a hyphen and suggest column width in the xml input, we're happy for suggestions.

Reply:
Hmmm.  I wasn't displeased by the ascii (txt) form of the table produced (by the author-tools site).  What do others think?  Given how important BGP (and QUIC) are to Internet technology, I'd hate to see the draft held up over something like this.

Reply:
Would you cache your rendered version with this change somewhere we can see?  I'm basing my opinion of this on my locally executed xml2rfc invocation.

Reply:
It's at https://github.com/gregbo/BGP-QUIC/blob/patch-1/draft-retana-idr-bgp-quic-02.txt

Reply:
Thanks for demonstrating that it's my local issue rather than an issue with the xml.  

Let's implement your changes here.

That said, please remove the .txt from the change set.  The authors would prefer to manage that artifact as part of our release process.

Reply:
done